internal/controller/postgrescluster/pgbouncer.go

```diff
@@ -103,6 +103,11 @@ func (r *Reconciler) reconcilePGBouncerConfigMap(
 		(feature.Enabled(ctx, feature.OpenTelemetryLogs) || feature.Enabled(ctx, feature.OpenTelemetryMetrics)) {
 		err = collector.AddToConfigMap(ctx, otelConfig, configmap)
 	}
+	// If OTel logging is enabled, add logrotate config
+	if err == nil && otelConfig != nil && feature.Enabled(ctx, feature.OpenTelemetryLogs) {
+		err = collector.AddLogrotateConfig(ctx, cluster.Spec.Instrumentation, configmap,
+			naming.PGBouncerFullLogPath, collector.PGBouncerPostRotateScript)
```

Comment:
🤔 Seeing a `collector` constant passed to a `collector` function has me wondering: Should this happen in `collector.NewConfigForPgBouncerPod` ?

Reply:
Hmm... Well right now `collector.NewConfigForPgBouncerPod` just creates the otel config, so I don't think this would go there (unless we changed it to return a separate `logrotate` config that then gets passed into `reconcilePGBouncerConfigMap` :thinking: )... Another option would be to have this happen in `collector.AddToConfigMap`, although then that function would have a bunch of `AddLogrotateConfig` calls for all the various components, but I guess that's not problematic? :thinking: